README.md

```diff
@@ -9,55 +9,41 @@ features.
 ## Contributing
 
 p2pmem-test is an active project. We will happily consider Pull
-Requests (PRs) submitted against the offical repo at
-www.github.com/sbates130272/p2pmem-test.git.
+Requests (PRs) submitted against the [official repo][1].
 
 ## Getting Started
+1. In order to run any P2P traffic, you'll need to have a (Linux based) OS
+that supports [p2pdma][2]. This module is available in all Linux kernels 4.20
+or newer, however it is NOT compiled in by default.
```

Comment:
Add "So you will almost certainly have to compile a kernel from source and install that. The instructions on how to do that are beyond the scope of this document but there is a tool that can help here (ref to my build-latest-p2pdma tool).

Reply:
I want to add this as a section as well.